R/utils.R

```diff
@@ -207,6 +209,18 @@ parse_database_error <- function(msg) {
   )
 }
 
+contextualize_database_error <- function(cnd_body) {
+  if (any(grepl("Data source name not found", cnd_body))) {
```

Comment:
Generally wary of `grep()`ping on messages, but from my understanding the unixODBC driver manager doesn't provide translations of its error messages to other languages.

Reply:
Yeah, and the failure mode isn't too bad here. Might be worth noting/checking that `cnd_body` is always length 1 and not missing.